docs/croissant-spec.md

```diff
@@ -368,7 +405,7 @@ mechanisms, or use `sc:PropertyValue` as an escape hatch.
 
 ### format
 
-A format used to parse the values coming from a `DataSource`, e.g., for dates or numbers.
+A format used to parse the values coming from a `DataSource` or a `BoundingBox`, e.g., for dates or numbers.
```

Comment:
Why single out Bounding Box? It's just a specific data type that  might be used for a data source, right ? Formats can apply to dates, numbers, bounding boxes, shape files, etc.